snap/hooks/install

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+# get the values of $SNAP_DATA and $SNAP using the current symlink instead of
+# the default behavior which has the revision hard-coded, which breaks after
+# a refresh
+SNAP_DATA_CURRENT=${SNAP_DATA/%$SNAP_REVISION/current}
+SNAP_CURRENT=${SNAP/%$SNAP_REVISION/current}
+
+# install all the config files from $SNAP/config/res/ into $SNAP_DATA/config,
+# but if files already exist, don't over-write
+mkdir -p "$SNAP_DATA/config"
+
+# create profile dirs if needed
+cd "$SNAP/config/"
+find . -maxdepth 2 -type d -exec mkdir -p "$SNAP_DATA/config/"{} \;
```

Comment:
:-( but understood why you need to do this

Reply:
It passed shellcheck...

Reply:
Yeah it's fine I'm just complaining about the unreadability of shellscript in general, nothing for you to worry about :smiley: